### PR TITLE
Move Client related models out of Accounts (OUT-95)

### PR DESCRIPTION
**Pull request by vecchp**
OUT-95

### CODE REVIEW

File: apps/betterangels-backend/notes/migrations/0007_alter_moodevent_id_alter_noteevent_id_and_more.py
Comment (vecchp) [inline apps/betterangels-backend/notes/migrations/0007_alter_moodevent_id_alter_noteevent_id_and_more.py:1]: I do not really understand where these `AlterField`'s' are coming from. @mikefeldberg maybe you have an idea

File: apps/betterangels-backend/accounts/migrations/0036_alter_userevent_id.py
Comment (vecchp) [inline apps/betterangels-backend/accounts/migrations/0036_alter_userevent_id.py:1]: I do not really understand where these `AlterField`'s' are coming from. @mikefeldberg maybe you have an idea

File: apps/betterangels-backend/notes/migrations/0007_alter_moodevent_id_alter_noteevent_id_and_more.py
Comment (vecchp) [reply apps/betterangels-backend/notes/migrations/0007_alter_moodevent_id_alter_noteevent_id_and_more.py:1]: This seems to be related to bumping django-pghistory

File: apps/betterangels-backend/accounts/migrations/0036_alter_userevent_id.py
Comment (vecchp) [reply apps/betterangels-backend/accounts/migrations/0036_alter_userevent_id.py:1]: This seems to be related to bumping django-pghistory

**Review by mikefeldberg — APPROVED** [review]
lgtm! thanks for making this happen 🙏

**Pull request merged**